plugins/win-capture/game-capture.c

```diff
@@ -1655,7 +1686,8 @@ static void game_capture_tick(void *data, float seconds)
 			debug("init_capture_data failed");
 
 		if (result != CAPTURE_RETRY && !gc->capturing) {
-			gc->retry_interval = ERROR_RETRY_INTERVAL;
+			gc->retry_interval = ERROR_RETRY_INTERVAL*
```

Comment:
Style: Typically, you need to surround math operators by a single space on each side. Don't leave a trailing space at the end of the line.